app/views/command-line.html

```diff
@@ -38,14 +38,22 @@ <h1 id="cli">Command Line Tools</h1>
                   <p>
                     After downloading and installing it, you can start by logging in using<span ng-if="sessionToken"> this current session token</span>:
                     <div class="code prettyprint ng-binding" ng-if="sessionToken">
```

Comment:
@juanvallejo need an outer div with `ng-if="sessionToken"` surrounding both the `prettyprint` div and the action links below.